scripts/build/Platform/Linux/build_config.json

```diff
@@ -58,6 +58,20 @@
       "CMAKE_TARGET": "all"
     }
   },
+  "profile_gcc_nounity": {
```

Comment:
any reason to do nounity? I would just run unity

Reply:
Memory concerns, gcc is a really poor performing compiler (see comment below)